src/test/resources/feature/WriteReview.feature

```diff
@@ -0,0 +1,17 @@
+@Irina
+Feature:   Write Review
+
+  Scenario: WriteReview - Positive
+    Given user is logged in
+    And "Home" is displayed
```

Comment:
We assume that after logging in Home page should be dispayed. We can assert that in first step

Reply:
'My Account' screen is displayed after signing in. The purpose of this step is that user 'navigates' to Home. Renamed the step